src/pages/careers/_join_your_team.js

```diff
@@ -0,0 +1,160 @@
+        },
+        {
+            img_data: data.operations,
+            to: 'careers/teams/operations/',
+            display_team_name: 'Operations',
+            tagline:
+                'We support all the teams in our organisation so that we’re able to achieve our goals without roadblocks and hiccups.',
+        },
+        {
+            img_data: data.customerSupport,
+            to: 'careers/teams/customer-support/',
+            display_team_name: 'Customer Support',
+            tagline:
+                'We care about providing helpful solutions to our clients to make their trading experience with us even better.',
+        },
+        {
+            img_data: data.recruitmentAndHR,
+            to: 'careers/teams/recruitment-hr/',
+            display_team_name: 'Recruitment & HR',
+            tagline:
+                'We bring in go-getters, link them with the right opportunities, and support them in every way to help them grow.',
+        },
+        {
+            img_data: data.productDesignAndDevelopment,
+            to: 'careers/teams/product-design-development/',
+            display_team_name: 'Product Design & Development',
+            tagline:
+                'We are the brains behind the creation of the platforms and products that drive the success of our business.',
+        },
+        {
+            img_data: data.compliance,
+            to: 'careers/teams/compliance/',
+            display_team_name: 'Compliance',
+            tagline:
+                'We navigate the legal and regulatory environment to help our business maintain the highest ethical standards.',
+        },
+        {
+            img_data: data.accountsAndPayments,
+            to: 'careers/teams/accounts-payments/',
+            display_team_name: 'Accounts & Payments',
+            tagline:
+                'We’re all about helping the company make good decisions about money so we can stay on course to fulfil our vision.',
+        },
+        {
+            img_data: data.paymentOperations,
+            to: 'careers/teams/payment-operations/',
+            display_team_name: 'Payment Operations',
+            tagline:
+                'We make it easy for clients to trade from anywhere in the world, with any currency. ',
+        },
+    ]
+
+    return (
+        <SectionContainer padding="12rem 0">
+            <Container direction="column">
+                <DiffHeader as="h3" type="section-title" align="center">
+                    Join Your Team?
+                </DiffHeader>
+            </Container>
+            <StyledCssGrid
+                columns="repeat(3, 384px)"
+                column_gap="2.4rem"
+                laptop_columns="repeat(3, 384px)"
+                laptop_row_gap="2.4rem"
+                tablet_columns="repeat(2, 300px)"
+                tablet_row_gap="2.4rem"
+                mobile_columns="90%"
+                mobile_row_gap="2.4rem"
```

Comment:
You might have a more flexible grid system if you use your own StyledGrid with a regular div with the following styles
```
grid-template-columns: repeat(auto-fit, minmax(288px, 384px));
grid-row-gap: 24px;
grid-column-gap: 24px;
```

Reply:
288px as the min is calculated from the smallest screen width we support being 320px with 16px padding each on the left & right (320px - 16px - 16px).

Reply:
You might want to look into the besquare banner again. The image disappears at about 900px ish.

Reply:
cool let me give it a try, thanks.